src/test/java/tests/SearchTest.java

```diff
@@ -20,4 +20,15 @@ public void searchTest() {
         for (WebElement element : productsPage.getAllProductsName())
             Assert.assertTrue(element.getText().contains(LAPTOP));
     }
+
+    @Test
+    public void searchTestWithClear() {
+        homePage.openHomePage();
+        homePage.inputSearchField();
+        homePage.searchButtonClick();
+        homePage.clear();
+
+        homePage.searchFieldGetText();
+        Assert.assertTrue(true);
```

Comment:
this test checks nothing. We can't check that true is true since it's always true....